screen-wake-lock/wakelock-onrelease.https.html

```diff
@@ -5,17 +5,19 @@
 <script src="/resources/testdriver.js"></script>
 <script src="/resources/testdriver-vendor.js"></script>
 <script>
-async_test(async t => {
+promise_test(async t => {
```

Comment:
Excellent, wakelock-onrelease.https.html fails fast with a clear error message in Firefox and Safari now!